Cargo.toml

```diff
@@ -25,6 +25,7 @@ alloc = ["rand_core/alloc"]  # enables Vec and Box support (without std)
 i128_support = [] # enables i128 and u128 support
 simd_support = [] # enables SIMD support
 serde1 = ["serde", "serde_derive", "rand_core/serde1"] # enables serialization for PRNGs
+rust_1_27 = [] # enables RangeInclusive support for Rust >= 1.27
```

Comment:
Sure, why not.

I think this can imply `i128_support`, but probably not `simd_support` since not all of that is stable yet(?).

Reply:
Good point. `simd_support` is currently broken and being addressed in #565.